ridgeback_bringup/launch/accessories.launch

```diff
@@ -6,7 +6,7 @@ in the URDF. See ridgeback_description/urdf/accessories.urdf.xacro.
 -->
 <launch>
   <!-- Primary LMS1xx-series LIDAR, base is front-facing but support for addtional rear. -->
-  <group if="$(optenv RIDGEBACK_FRONT_LASER 0)">
+  <group if="$(optenv RIDGEBACK_FRONT_LASER_TYPE sick_lms1xx)">
```

Comment:
You'd want to check both, wouldn't you? eg, a big `<group>` for "Is the front laser enabled?" and then two internal ones for "do I launch driver x or driver y?"